cli/cmd/plugin/ui/web/tanzu-ui/package.json

```diff
@@ -33,7 +33,7 @@
     "start": "npm run lint && npm run generate-api && GENERATE_SOURCEMAP=false react-scripts start",
     "build": "npm run lint && npm run generate-api && GENERATE_SOURCEMAP=false react-scripts build",
     "build:prod": "npm ci && npm run build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --transformIgnorePatterns \"node_modules/(?!ui-core)/\" --env=jsdom",
```

Comment:
I don't know why the existing one is working, but I think this one `"test": "react-scripts test --transformIgnorePatterns \"node_modules/(?!@cds)/\"",` should make more sense.  This means that it will not transform files inside the node_modules directory except for these in the @cds directory